db/migrate/20230927094602_alter_journals.rb

```diff
@@ -0,0 +1,13 @@
+class AlterJournals < ActiveRecord::Migration[7.0]
+  def up
+    add_column :journals, :user_id, :integer, index: true, foreign_key: true
```

Comment:
Can you try this and test if all good ?
```suggestion
    add_reference :journals, :user, null: false, foreign_key: true
```